common/remote/hooks/azure/azurefile.cpp

```diff
@@ -964,16 +963,20 @@ class AzureFileClient : public AzureAPICopyClientBase
     {
         fileClient->Delete();
     }
+public:
+    AzureFileClient(const char *target)
```

Comment:
It makes more sense to provide 'target' in the constructor as the class is associated with the target.  i.e. AzureFileClient requires target to always be an azure file uri.